draft-ietf-tls-tls13.md

```diff
@@ -3042,18 +3042,23 @@ This document defines one ticket extension, "ticket_early_data_info"
 ~~~~
    struct {
        uint32 ticket_age_add;
+       uint32 max_early_data_size;
    } TicketEarlyDataInfo;
 ~~~~
 
 This extension indicates that the ticket may be used to send 0-RTT data
-({{early-data-indication}})). It contains one value:
+({{early-data-indication}})). It contains two values:
```

Comment:
(editorial nit: do we really need to have the number here?  Could it just be "Its contents are:"?)